### DISCUSSION TITLE
Haven't Received GitHub Education Benefits After Approval

### DISCUSSION THREAD

**Discussion by dipakrathod-tech** [community/community#139579] [GitHub Education]
### Hi everyone,

I was approved for GitHub Education benefits on [insert date], but I have not received them yet. It's been more than 5 days, and I'm seeking assistance.

I’ve checked my account settings and haven’t found any updates. Is there anything else I should check or do?

Any guidance on resolving this issue would be greatly appreciated!
![Screenshot 2024-09-24 135241](https://github.com/user-attachments/assets/67034ef8-1f49-4493-a780-7cab99ab6668)

**Comment by davevad93** [comment]
Hi @Mr-Dipak ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Mr-Dipak was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by dipakrathod-tech** [reply]
hey @davevad93 Thank you for you precious guidance, Thank you once again!

**Reply by davevad93** [reply]
You're welcome @Mr-Dipak 👍
